apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -142,6 +144,9 @@ class ApplabVisualizationColumn extends React.Component {
         <GameButtons noRunResetButton={playspacePhoneFrame}>
           {/* This div is used to control whether or not our finish button is centered*/}
           <div style={this.getCompletionButtonSyle()}>
+            {experiments.isEnabled(experiments.CSP_VALIDATION_VIA_AI) && (
+              <ValidationButton />
+            )}
```

Comment:
is this the only thing the users needs in order to use tutor in this way? or do they also need the `'ai-tutor'` experiment?

Reply:
For someone to have [access to AI Tutor,](https://github.com/code-dot-org/code-dot-org/blob/3781f57438c8642ad998b5b480fa75742916c2f3/dashboard/app/models/user.rb#L1582) they either need to have the ai tutor access permission or be in the section of a teacher enrolled in the ai tutor experiment. 

Rather than doling out permissions to folks to test internally, my plan is to set up a section as a teacher in the ai tutor experiment and then have Tess, Erik, and any engineers who want to try things out join that section. This worked well for AI Tutor.